tests/sentry/api/endpoints/test_organization_monitor_details.py

```diff
@@ -44,6 +44,16 @@ def test_name(self):
         monitor = Monitor.objects.get(id=monitor.id)
         assert monitor.name == "Monitor Name"
 
+    def test_slug(self):
```

Comment:
worth adding a test where `slug` is specified but is `None` or `""`, I don't think the regex checker will allow it in `MonitorValidator`. generally we allow `None` values as equivalent to not specifying an optional param

Reply:
Updated. It fails with 'cannot be blank'